pkg/common/cns-lib/volume/util.go

```diff
@@ -547,19 +547,10 @@ func queryCreatedSnapshotByName(ctx context.Context, m *defaultManager, volumeID
 	return nil, false
 }
 
-// IsNotFoundError returns true if a given faulType is of type NotFound
-func IsNotFoundError(faultType string) bool {
-	// Extract error name from faultType
-	faultTypeErrSplitString := strings.Split(faultType, ".")
-	faultTypeErr := faultTypeErrSplitString[len(faultTypeErrSplitString)-1]
-
-	// Get NotFound error name from CNS vim25 types.
-	notFoundErrSplitString := strings.Split(((reflect.TypeOf(types.NotFound{})).String()), ".")
-	// vim25types gives value of format *type.XXX.
-	// In this case it will be *type.NotFound
-	// Extract Notfound from the vim25types error.
-	notFoundErr := notFoundErrSplitString[len(notFoundErrSplitString)-1]
-
-	// Check weather the faultTye is NotFound
-	return notFoundErr == faultTypeErr
+// IsNotFoundFault returns true if a given faulType is of type vim.fault.NotFound
```

Comment:
`// IsNotFoundFault returns true if a given faulType is of type vim.fault.NotFound`
faulType ==> faultType

Can be rephrased to `// IsNotFoundFault returns true if a given faultType value is vim.fault.NotFound` since it is checking for string value.

Reply:
done.